src/materialized/src/bin/materialized/main.rs

```diff
@@ -377,6 +377,9 @@ pub struct Args {
     /// Turn on the console-subscriber to use materialize with `tokio-console`
     #[clap(long, hide = true)]
     tokio_console: bool,
+
+    #[clap(long)]
```

Comment:
Please add `hide = true` here.

Reply:
Oops. I fixed this and then force pushed over your test fix up. Sorry about that.